<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuickMenu</class>
 <widget class="QDialog" name="QuickMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>120</width>
    <height>121</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Quick Menu</string>
  </property>
  <widget class="QToolButton" name="strokeEraserButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>41</width>
     <height>42</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../myresource.qrc">
     <normaloff>:/images/strokeEraserIcon.png</normaloff>:/images/strokeEraserIcon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="selectButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>41</width>
     <height>42</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../myresource.qrc">
     <normaloff>:/images/selectIcon.png</normaloff>:/images/selectIcon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="rectSelectButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>41</width>
     <height>42</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../myresource.qrc">
     <normaloff>:/images/rectSelectIcon_48.png</normaloff>:/images/rectSelectIcon_48.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="handButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>41</width>
     <height>42</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../myresource.qrc">
     <normaloff>:/images/handIcon.png</normaloff>:/images/handIcon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="penButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>41</width>
     <height>42</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../myresource.qrc">
     <normaloff>:/images/penIcon.png</normaloff>:/images/penIcon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="circleButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>0</y>
     <width>41</width>
     <height>42</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../myresource.qrc">
     <normaloff>:/images/circleIcon.png</normaloff>:/images/circleIcon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="eraserButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>41</width>
     <height>42</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../myresource.qrc">
     <normaloff>:/images/eraserIcon.png</normaloff>:/images/eraserIcon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="rectButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>41</width>
     <height>42</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../myresource.qrc">
     <normaloff>:/images/rectIcon_48.png</normaloff>:/images/rectIcon_48.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="rulerButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>41</width>
     <height>42</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../myresource.qrc">
     <normaloff>:/images/rulerIcon.png</normaloff>:/images/rulerIcon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../myresource.qrc"/>
 </resources>
 <connections/>
 <slots>
  <signal>pen()</signal>
 </slots>
</ui>
